.env.example

```diff
@@ -1,17 +1,15 @@
-# The format:
-# SPOTIFY_SECRETS=clintId1:clientSecret1,clientId2:clientSecret2
-SPOTIFY_SECRETS=$SPOTIFY_SECRETS
+# Structure: client,secret:client,secret
+SPOTIFY_SECRETS=
 
-# 0 or 1
-# 0 = disable
-# 1 = enable
-ENABLE_UPDATE_CHECK=$ENABLE_UPDATE_CHECK
+# Choice: true (enter: 1) or false (enter: 0)
+ENABLE_UPDATE_CHECK=
 
-LASTFM_API_KEY=$LASTFM_API_KEY
-LASTFM_API_SECRET=$LASTFM_API_SECRET
+LASTFM_API_KEY=
+LASTFM_API_SECRET=
 
-# Release channel. Can be: nightly, stable
-RELEASE_CHANNEL=$RELEASE_CHANNEL
+# Choice: stable or nightly
+RELEASE_CHANNEL=
 
-HIDE_DONATIONS=$HIDE_DONATIONS
-DISABLE_SPOTIFY_IMAGES=$DISABLE_SPOTIFY_IMAGES
```

Comment:
There's a reason why we need right side of the equal sign. `envsubst` requires that in the CI. Revert changes here